unikraft/lib/template/template.go

```diff
@@ -0,0 +1,262 @@
+package template
+
+import (
+	"context"
+	"html/template"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	_ "embed"
+
+	git "github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/go-git/go-git/v5/plumbing/object"
+)
+
+var (
+	//go:embed CODING_STYLE.md.tmpl
+	CodingStyleTemplate string
+
+	//go:embed Config.uk.tmpl
+	ConfigUkTemplate string
+
+	//go:embed CONTRIBUTING.md.tmpl
+	ContributingTemplate string
+
+	//go:embed COPYING.md.tmpl
+	CopyingTemplate string
+
+	//go:embed main.c.tmpl
+	MainTemplate string
+
+	//go:embed Makefile.uk.tmpl
+	MakefileUkTemplate string
+
+	//go:embed README.md.tmpl
+	ReadmeTemplate string
+)
+
+type Template struct {
```

Comment:
I would rename this to `LibTemplate` as it can be confused to the existing application templates

Reply:
Done 👍🏽